django/contrib/contenttypes/checks.py

```diff
@@ -5,7 +5,7 @@
 
 
 def check_generic_foreign_keys(app_configs=None, **kwargs):
```

Comment:
PR #17886 attempts to remove this check function, though I haven't managed to get that into a mergeable state yet.